stdlib/3/dis.pyi

```diff
@@ -0,0 +1,70 @@
+from typing import Any, List, Union, NamedTuple, Iterator, Tuple, IO, Optional, Dict
+
+from opcode import cmp_op,hasconst ,hasname , hasjrel, hasjabs, haslocal, hascompare, hasfree, opname ,opmap ,HAVE_ARGUMENT, EXTENDED_ARG, hasnargs
```

Comment:
Please fix the bad space/comma usage on this line.

Would also be a good idea to make the lines shorter, e.g. staying within 100 chars.